src/CommunityToolkit.Maui.CameraView/CameraInfo.cs

```diff
@@ -0,0 +1,80 @@
+﻿using CommunityToolkit.Maui.Core.Primitives;
+using CommunityToolkit.Mvvm.ComponentModel;
+using System.Collections.ObjectModel;
+
+#if IOS || MACCATALYST
+using AVFoundation;
+#elif ANDROID
+using AndroidX.Camera.Core;
+#elif WINDOWS
+using Windows.Media.Capture.Frames;
+using Windows.Media.MediaProperties;
+#endif
+
+namespace CommunityToolkit.Maui.Core;
+
+public partial class CameraInfo : ObservableObject
```

Comment:
As mentioned in the project file, I don't think we should be bringing in a dependency on the MVVM Toolkit.

Out of interest though, does this info really need to change? I expect that this could be immutable as it represents a physical camera?

Reply:
The reason for making it observable is on Windows some camera properties such as zoom factors can only be queried after initializing a `MediaCapture` object, which is done in the `CameraManager` after starting the view. Doing so in `CameraProvider` where the `CameraInfo` is initialized is not desirable as this class could be used without starting the CameraView.